package.nls.json

```diff
@@ -419,6 +419,7 @@
     "AWS.samcli.deploy.general.error": "An error occurred while deploying a SAM Application. {0}",
     "AWS.samcli.deploy.workflow.init": "Building SAM Application...",
     "AWS.samcli.deploy.workflow.packaging": "Packaging SAM Application to S3 Bucket: {0}",
+    "AWS.samcli.deploy.workflow.packaging.noBuild": "Build unsuccessful. AWS Toolkit will attempt to package source template directory directly.",
```

Comment:
I suggested the wording `Attempting SAM deploy after "sam build" failed:\nline 1\nline 2\n...` because "build failed" is not the main point and will unnecessarily alarm users. It is mostly informational.

Reply:
We currently output to the channel log. I can pull this info if you'd like...does this make it less threatening? (might be less alarming to just link users to the channel in the toast)

Also, keep in mind: non-modal notifications don't allow for `\n`. I don't think this justifies a modal unless we want users to decide whether or not to proceed with the operation.

Reply:
The minimum I'm suggesting is the specific wording, i.e. start with "Attempting SAM deploy after..." instead of "Build unsuccessful"

Reply:
FYI, this is the output from what we consider the expected case (Lambda created in console, then pulled down into Cloud9 via old toolkit, then pulled into the new extension and deploy attempted with `AWS: Deploy SAM Application` command) with the newline characters replaced; this comes straight from the SAM process.
```
Error with child process: 2021-01-21 11:58:14,397 | Telemetry endpoint configured to be https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics
,2021-01-21 11:58:15,024 | 'build' command is called
,2021-01-21 11:58:15,026 | No Parameters detected in the template
,2021-01-21 11:58:15,057 | 1 resources found in the template
2021-01-21 11:58:15,058 | Found Serverless function with name='madeInConsole' and CodeUri='.'
,2021-01-21 11:58:15,058 | No Parameters detected in the temp…49291345db3', 'executionEnvironment': 'CLI', 'pyversion': '3.8.6', 'samcliVersion': '1.15.0'}}]}
,2021-01-21 11:58:17,058 | Telemetry response: 200
,Error: NodejsNpmBuilder:NpmPack - NPM Failed: npm ERR! code ENOLOCAL
npm ERR! Could not install from "..\\..\\..\\..\\testApps\\test12-10\\madeInConsole" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\\Users\\xxxxxxxx\\AppData\\Roaming\\npm-cache\\_logs\\2021-01-21T19_58_16_872Z-debug.log
```

Reply:
Yes we would need to use a filter to get 1-2 lines that are actually relevant. I don't see anything relevant in that output :P